core/templates/dev/head/pages/exploration_editor/issues_tab/issues_tab.html

```diff
@@ -0,0 +1,31 @@
+<div ng-controller="IssuesTab"
+     class="oppia-dashboard-container issues-tab-dashboard" style="position: relative;">
+  <div class="oppia-dashboard-aggregated-stats" style="margin: auto;">
+    <md-card layout="row">
+      <div class="open-issues stats-card">
+        <!-- TODO: Add translations -->
```

Comment:
Can these translation TODOs be resolved before this PR is submitted? Prefer not hardcoding English strings directly.

Reply:
Hmm, I guess it depends on how hard it is to get all the languages supported? Would it be OK to only have english working since it's still WIP?

Reply:
It's not too hard... You would need to add the text to be translated into en.json, and add a description for the text in qqq.json. And then use the corresponding i18n variable in the code. But I leave the decision to you on whether you want to do it in this PR or in a later one.

Reply:
OK, done.